components/common/AppLogo.vue

```diff
@@ -1,27 +1,17 @@
 <script setup lang="ts">
-import { EXTERNAL_LOGO } from '~/utils/external-links';
-
-withDefaults(
-  defineProps<{
-    src?: string;
-    text?: boolean;
-  }>(),
-  {
-    src: EXTERNAL_LOGO,
-    text: false,
-  },
-);
+withDefaults(defineProps<{ text?: boolean }>(), { text: false });
 
 const attrs = useAttrs();
+const branch = process.env.NODE_ENV === 'development' ? 'develop' : 'main';
```

Comment:
We should use the `useRuntimeConfig` composable here instead of `process.env.NODE_ENV`.

We should already be using the following `NUXT_PUBLIC_TESTING` to switch between production/testing so `public.testing` from runtime config should help is figure out the branch